db/migrate/20220209165604_add_rhc_connections.rb

```diff
@@ -0,0 +1,25 @@
+class AddRhcConnections < ActiveRecord::Migration[5.2]
+  def up
+    create_table :rhc_connections do |t|
+      t.string :rhc_id
+      t.jsonb :extra, :default => '{}'
+      t.string :availability_status
+      t.string :availability_status_error
```

Comment:
do we need also  `last_checked_at` or `last_available_at` ?

Reply:
ah yeah completely spaced those. will add them.